experimental/dds/tree2/src/events/events.ts

```diff
@@ -198,7 +198,7 @@ export class EventEmitter<E extends Events<E>> implements ISubscribable<E>, HasL
 	private readonly listeners = new Map<keyof E, Set<(...args: unknown[]) => any>>();
 
 	// Because this is protected and not public, calling this externally (not from a subclass) makes sending events to the constructed instance impossible.
-	// Instead, use the static `create` function to get an instance which allows emitting events.
+	// Instead, use the `createEmitter` function to get an instance which allows emitting events.
```

Comment:
No static `create` function exists on `EventEmitter`, I imagine we moved to `createEmitter` at some point but forgot to update the comment.